corehq/apps/registration/forms.py

```diff
@@ -46,9 +46,6 @@ class NewWebUserRegistrationForm(DomainRegistrationForm):
     """
     Form for a brand new user, before they've created a domain or done anything on CommCare HQ.
     """
-    full_name = forms.CharField(label=_('Full Name'),
```

Comment:
Put full name back. How strongly do you feel that it should be optional? As long as the input is there, I feel like we might as well leave it required, unless there's some set of users who are opposed to entering their names...?